Directory.Build.props

```diff
@@ -45,9 +46,7 @@
 
     <!-- Output directories -->
     <BinDir Condition="'$(BinDir)'==''">$(ProjectDir)bin/</BinDir>
-
-    <ObjDir Condition="'$(ObjDir)'==''">$(BinDir)obj/</ObjDir>
```

Comment:
Makes me a bit worried the fact that we are not defining this property anymore. I wonder if there will be any side effects like stuff binplaced now on the source tree.

Reply:
ObjDir isn't used anywhere in buildtools